### ISSUE TITLE
[Misdetection] Eicar.txt file misdetected as text/vbscript

### ISSUE THREAD

**Issue by Vajaylm** [google/magika#705]
Eicar.txt is misdetected as `text/vbscript` instead of `text/plain` when using python-v0.5.0 or python-v0.5.1 release. Using the demo on [https://google.github.io/magika](https://google.github.io/magika), the detection is correct as it is `text/plain`. 

The content of the file can be reached on this link: [https://secure.eicar.org/eicar.com.txt](https://secure.eicar.org/eicar.com.txt)

Could you please help to fix this so the detection can be correct from a Python script?

Thank you in advance!

**Comment by Vajaylm** [comment]
Hi @reyammer,
Are there any progress regarding this issue? Were you be able to reproduce it and investigate?
Thank you in advance!

**Comment by Vajaylm** [comment]
Hi @reyammer 
As I got no update in the last 4 month, I ask again if you were able to reproduce or make any progress regarding the issue. 
Thank you in advance!

**Comment by reyammer** [comment]
Hello @Vajaylm, apologies for the silence! Too much on the plate at the moment, sorry!

A couple of comments:
- I still don't know why the CLI and the tfjs version differ. Can it be that the python client says "vba" with low confidence, and the tfjs implementation actually uses that info and outputs plain text instead of vba? What's the confidence score you are getting?
- unfortunately I'm not too familiar with the tfjs version and debugging is tricky. But figuring out this bug is somehow losing relevance as we are very near to push a new model out (see https://github.com/google/magika/pull/866); for the new version we have much stronger consistency tests, and this bug will likely be addressed when we port the javascript version to use the new model as well. /CC @invernizzi 
- About using magika to detect the EICAR file: I agree it would be ideal to detect is a plaintext, but in the eyes of the model, that's a weird-looking string (the latest model detects it as powershell... and I can't really blame it!). And it's tricky for us to fix specific one-off false positives such as this one. We are able to fix "patterns of false positives" by improving the dataset or the model, but this one is really a one-off. And for such popular one-offs, I believe that using ML is an overkill; if detecting that string reliably is relevant for your case, I suggest you add an explicit `if` to check for it before using the model.

Hope it helps!

(I'm leaving this open in case someone has cycles to check deeper.)

**Comment by invernizzi** [comment]
I agree with Yanick: this is really a _special_ case, as it is intentionally designed to be odd looking.

It has a `.txt` extension, but it's also provided with a `.com` [extension](https://www.eicar.org/download-anti-malware-testfile/) with exactly the same content. It used to be called `ducklin.htm` too. Because of that, Magika cannot be right, though I agree it could be right-er than saying `powershell`.

These special cases should be handled with an override list, either project specific or community-driven. That is easier to manage and customize, and we don't risk confusing the model during training.
